sockutils.c

```diff
@@ -794,7 +819,9 @@ int sock_send(SOCKET sock, SSL *ssl _U_NOSSL_, const char *buffer, size_t size,
 		if (ssl) return ssl_send(ssl, buffer, remaining, errbuf, errbuflen);
 #endif
 
-#ifdef MSG_NOSIGNAL
```

Comment:
Same here.

Reply:
Not sure it is relevant here, but doing it